test/integration/test.speech_to_text.js

```diff
@@ -380,5 +380,42 @@ describe('speech_to_text_integration', function() {
       speech_to_text.deleteCustomization({ customization_id: customization_id }, done);
       customization_id = null;
     });
+
+    describe('asynchronous api', function() {
+      let jobId = null;
+
+      it('registerCallback()', function(done) {
+        const params = { callback_url: 'http://watson-test-resources.mybluemix.net/results', user_secret: 'ThisIsMySecret' };
+        speech_to_text.registerCallback(params, done);
+      });
+
+      it('createRecognitionJob()', function(done) {
+        const params = {
+          audio: fs.createReadStream(__dirname + '/../resources/weather.wav'),
+          content_type: 'audio/l16;rate=41100',
+          callback_url: 'http://watson-test-resources.mybluemix.net/results',
+          user_token: 'ThisIsMySecret',
```

Comment:
Please change this to something like `'my-arbitrary-identifier-1'`.

`uset_token` is an optional arbitrary string that you can set to easily tie the event callbacks to this original request. However, it should *not* be set to the same value as your secret, because that would compromise the secret.